### PR TITLE
Update `Run` intro page with calls to action

### PR DESCRIPTION
**Pull request by kaelynj**
Realized the introduction page of the `Run` section is fairly sparse and decided to add in a *Next Steps* section similar to the other pages with a few calls to action.

### CODE REVIEW

File: docs/run/index.mdx [hunk 1]
```diff
@@ -19,4 +19,16 @@ The steps during the run phase are:
...
 
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
+  -  Understand the [primitives interface](./primitives)
+  -  Learn about the [error suppression and mitigation options](./configure-error-mitigation) available to you through Qiskit Runtime.
+  -  Read more about our [hardware platform](./processor-types)
+  -  Understand [execution modes](./execution-modes) and [job management](./save-jobs)
+</Admonition>
```
Comment (abbycross) [inline docs/run/index.mdx:30]:
```suggestion
  -  Understand the [primitives interface](./primitives).
  -  Learn about the [error suppression and mitigation options](./configure-error-mitigation) available to you through Qiskit Runtime.
  -  Read more about [processor types](./processor-types).
  -  Understand [execution modes](./execution-modes) and [job management](./save-jobs).
</Admonition>
```

**Comment by Eric-Arellano** [comment]
This is good to merge, although needs to fix merge conflicts

**Comment by abbycross** [comment]
I think (correct me if I'm wrong??) this page was overwritten by [Execute on target hardware](https://docs.quantum.ibm.com/guides/execute-on-hardware) in the reorg - and if so, we can close this PR

**Comment by Eric-Arellano** [comment]
Ah, indeed, it was.

**Pull request closed without merging**